util/theme.js

```diff
@@ -7,6 +7,19 @@ const offsetDotSvgUrl =
 const offsetRoundedDotSvgUrl =
   'url(\'data:image/svg+xml;utf8,<svg height="15" viewBox="0 0 10 15" width="10" xmlns="http://www.w3.org/2000/svg"><g fill="%23000" fill-rule="evenodd"><rect height="5" rx="1" width="5"/><rect height="5" rx="1" width="5" x="5" y="7.5"/></g></svg>\')'
 
+export const defaultPointerUrl =
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Curious... what was prettier complaining about here?

Reply:
yes, eslint and prettier was fighting against each other

Reply:
Oh I hate that.